roles/cross_seed/defaults/main.yml

```diff
@@ -64,7 +65,8 @@ cross_seed_docker_commands: "{{ cross_seed_docker_commands_default
 # Volumes
 cross_seed_docker_volumes_default:
   - "{{ cross_seed_paths_location }}:/config"
-  - "/mnt/local/downloads/torrents:/torrents"
+  - "/opt:/opt"
```

Comment:
Should be `:ro`

Reply:
Yes, forgot to add that, just a second here